supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -2,6 +2,18 @@
 [[code-examples]]
 = Code examples
 
+Provide the source language if it is supported by the toolchain. Do not use the Bash source language for terminal commands. Bash improperly interprets the number sign (#) as a comment instead of the prompt for a command run as the root user.
+
+.Example Asciidoc
+
+[source, yaml]
```

Comment:
You can't actually see this [source,yaml] since it's rendering. You should be able to indent the entire example by two spaces and have it all render in a code block.

Reply:
Also, not critical, but I don't think there's supposed to be a space after the comma, even though it renders fine.